docs/conf.py

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
+#
+# PACKAGENAME documentation build configuration file, created by
+# sphinx-quickstart on Wed Jul  5 16:25:26 2017.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+"""Configuration script for Sphinx documentation."""
+
+import json
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.autodoc',
+              'sphinx.ext.intersphinx',
+              'sphinx.ext.todo',
+              'sphinx.ext.coverage',
+              'sphinx.ext.mathjax',
+              'sphinx.ext.ifconfig',
+              'sphinx.ext.viewcode',
+              'sphinx.ext.githubpages',
+              'numpydoc',
+              'm2r2']
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['.templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The main toctree document (using required variable name).
+master_doc = 'index'
+
+# General information about the project.
+project = 'PACKAGENAME'
+title = '{:s} Documentation'.format(project)
+zenodo = json.loads(open('../.zenodo.json').read())
+author = ', '.join([auth['name'] for auth in zenodo['creators']])
+description = 'PACKAGENAME DESCRIPTION.'
+category = 'PACKAGENAME CATEGORY'  # e.g., 'Space Physics'
+copyright = ', '.join(['YYYY', author])
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+doc_dir = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(doc_dir, "..", project, "version.txt"), "r") as fin:
+    version = fin.read().strip()
+release = '{:s}-alpha'.format(version)  # Include alpha/beta/rc tags.
+
+# The language for content autogenerated by Sphinx. Refer to documentation
+# for a list of supported languages.
+#
+# This is also used if you do content translation via gettext catalogs.
+# Usually you set "language" from the command line for these cases.
+language = None
```

Comment:
```suggestion
language ='en'
```
`None` is deprecated here